src/components/EditOrgsForm.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <div class="flex flex-column gap-3">
+    <div class="form-container">
+      <div class="form-field w-full">
+        <label :class="{ 'font-light uppercase text-sm': !editMode }">Name</label>
+        <div v-if="!editMode" :class="{ 'text-xl': !editMode }">{{ serverOrgData.name ?? 'None' }}</div>
+        <PvInputText v-else v-model="localOrgData.name" />
+      </div>
+      <div class="form-field w-full">
+        <label :class="{ 'font-light uppercase text-sm': !editMode }">Abbreviation</label>
+        <div v-if="!editMode" :class="{ 'text-xl': !editMode }">{{ serverOrgData.abbreviation ?? 'None' }}</div>
+        <PvInputText v-else v-model="localOrgData.abbreviation" />
+      </div>
+    </div>
```

Comment:
To avoid repetitions and design inconsistencies, what do you think of creating a general `TextInput` component that can be used here and in all other components that include a form? Ideally we would expand this to have `<FormContainer>`, `<TextInput>`, `<Checkbox>`, `<FormSubmit>` (which leverages a `<Button>`), etc.

This would primarily facilitate creating new and updating existing forms as well as future design changes. At the moment, in the existing approach, we would have to update the tailwind classes in all the different components that use a text input instead of updating them in a single place

Reply:
I like this approach. I'm inclined to add this as a separate task rather than hold up this current PR. What do you think @kellyel and @maximilianoertel?

Reply:
Definitely something that could (and really should) be done as a separate PR, happy to write a ticket for it as I was planing on to do so any way. Let me know!